src/ir/item.rs

```diff
@@ -825,6 +826,50 @@ impl Item {
             _ => None,
         }
     }
+
+    /// Returns whether the item is a constified module enum
+    fn is_constified_enum_module(&self, ctx: &BindgenContext) -> bool {
```

Comment:
What about modifying `ItemResolver` to optionally disable recursive resolving? More generally, `ItemResolver` could even take a positive integer indicating through how many layers to resolve.

This way, `is_constified_enum_module()` could be simplified to use `ItemResolver`.